### DISCUSSION TITLE
GOG auto-claim URL

### DISCUSSION THREAD

**Discussion by ALERTua** [vogler/free-games-claimer#242] [Ideas]
A Lemmy [post](https://sh.itjust.works/post/8103619) got me an idea to suggest its solution to GOG free games autoclaiming, using this URL:
https://www.gog.com/giveaway/claim

> It will give a json response. Visiting a second time should return “Already claimed” if it worked.

**Comment by mgrimace** [comment]
I was thinking the same, I’ve been getting some timeout errors and captcha errors with GOG claiming on the current script. The URL worked as expected. Visiting a second time generates the confirmation ‘already claimed’ message. 

Probably something like:
- Login script
- Visit auto claim URL
- Confirm by visiting auto claim URL a second time (see ‘already claimed’ message)
- If it works (gets ‘already claimed’), wait and just re-visit the URL every day twice daily for confirmation
- If it doesn’t work, proceed to the ‘old’ method?

**Reply by ALERTua** [reply]
> I’ve been getting some timeout errors and captcha errors with GOG claiming on the current script

same.

**Reply by vogler** [reply]
https://github.com/vogler/free-games-claimer/issues/240#issuecomment-1803035146

**Comment by ALERTua** [comment]
> [#240 (comment)](https://github.com/vogler/free-games-claimer/issues/240#issuecomment-1803035146)

ooh. thank you! closing this.
